<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmQHLoginDlg</class>
 <widget class="QDialog" name="frmQHLoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmQHLoginDlg</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>491</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>430</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
          <property name="text">
           <string>期货账户登陆</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_close">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-color: rgb(223, 223, 223);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>请输入客户号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-color: rgb(223, 223, 223);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>请输入密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-color: rgb(223, 223, 223);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>请输入Broke Id</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_5">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>记住密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_6">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>自动登录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="pushButton_login">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>登陆</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_exit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
